cluster-provision/images/vm-image-builder/alpine-cloud-init/create-image.sh

```diff
@@ -1,18 +1,24 @@
 #!/bin/sh
 set -ex
 
+function detect_cri() {
+    if podman ps >/dev/null 2>&1; then echo "using podman"; elif docker ps >/dev/null 2>&1; then echo "using docker"; fi
```

Comment:
you need to only echo podman or docker, without using, as this will be the binary name
same for the rest please

it wont work this way

Reply:
Done